python3.html.markdown

```diff
@@ -38,7 +38,7 @@ Note: This article applies to Python 3 specifically. Check out [here](http://lea
 8 - 1   # => 7
 10 * 2  # => 20
 
-# Except division which returns floats, real numbers, by default
+# Except division which returns integers if operating on integers
```

Comment:
5/2 returns 2, since 5 and 2 are both integers. old line 41 doesn't make any sense and i don't know what it's trying to say. 

what i'm saying is that if division operates on 5 and 2 (both integers), it returns a rounded number (2)